save-cloud-charts/save-cloud/templates/demo-agent-service.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: demo-agent-service
+spec:
+  selector:
+    io.kompose.service: save-demo-agent
+  clusterIP: None
+  ports:
+    - port: 23456
```

Comment:
Is this matching specific port of agent? Could be extracted into `values.yaml` too

Reply:
or migrate to default 80 port for example

Reply:
Implemented the first solution